test/format/config.sh

```diff
@@ -273,6 +277,8 @@ CONFIG configuration_list[] = {
 
 {"random.extra_seed", "set random seed for extra operations", 0x0, 0, 0, UINT_MAX}
 
+{"random.thread_seed", "set thread-local random seed", 0x0, 0, 0, UINT_MAX}
```

Comment:
I think this is ok, but just in case I'd like to confirm this won't impact on predictable replay. CC @sulabhM

Reply:
I intentionally added a RNG that's guaranteed to be unique across threads and re-playable, no matter if `format.independent_thread_rng` is set or not.
Other existing RNGs unfortunately don't have these properties.